sam-hiberry-app/src/orders/errors/base_error.py

```diff
@@ -0,0 +1,38 @@
+
+# Python's Libraries
+import logging
+
+
+class BaseError(Exception):
+
+    def __init__(self, _message, _error=None, _logger=None, _source=None,):
```

Comment:
Not sure about _ underscore in parameter names . They suggest private variables, but I'm confused in the context of a public constructor.